integrations/optimizely/lib/index.js

```diff
@@ -22,7 +22,8 @@ var Optimizely = (module.exports = integration('Optimizely')
   .option('variations', false) // send data via `.identify()`
   .option('listen', true) // send data via `.track()`
   .option('nonInteraction', false)
-  .option('sendRevenueOnlyForOrderCompleted', true));
+  .option('sendRevenueOnlyForOrderCompleted', true)
+  .option('customProps', {}));
```

Comment:
Would be possible to call it `customExperimentProperties`? Since this only affects to the `Experiment Viewed` event sent to Segment.